### DISCUSSION TITLE
Problem in Creating Application on HCP Cluster

### DISCUSSION THREAD

**Discussion by sSajjad90** [idaholab/moose#26467] [Q&A Getting Started]
Hi,
I encountered an issue while creating an application on the ComputeCanada HCP cluster. Following the installation of MOOSE on the cluster, including PETSc, libMesh, and WASP components, I successfully built and tested MOOSE as per the provided instructions. The tests concluded with `3839 passed, 367 skipped, 0 pending, and 0 failed`. Subsequently, I proceeded to create an application named "precipitation" (refer to the attached picture). After initial successful make and test operations, I modified the `MakeFile`, changing "All Module" to "Yes." However, this adjustment did not yield the expected results. It seems that only one test was rechecked instead of the anticipated comprehensive check of multiple tests. 
I suspect there might be an issue with the testing process or the MakeFile configuration. Could you kindly provide insights into what might be causing this discrepancy and guide me on resolving the problem?

Many thanks in advance for your kind support,
Sajjad

PS: I tested both `make -j 6` and `make -j 1` while creating and testing, and the results were same.

![Screenshot from 2024-01-03 12-03-36](https://github.com/idaholab/moose/assets/78385504/9ff32686-526d-4f1f-93c7-c87a03457a4e)

![Screenshot from 2024-01-03 12-29-19](https://github.com/idaholab/moose/assets/78385504/bad2c4f3-ffc2-4266-9616-061b47789748)

**Comment by GiudGiud** [comment]
Hello 

When you create a new application it comes preloaded with only a single test as an example. 

If you want to you can pass arguments to run_tests (see --help) to run the moose framework or modules tests with your application but it s not necessary. 

Guillaume
